backend/resilience/models.py

```
@@ -0,0 +1,226 @@
+    AbstractBaseModel,
+)
+from django.db.models import (
+    BooleanField,
+    CharField,
+    FloatField,
+    ForeignKey,
+    IntegerField,
+    ManyToManyField,
+    TextField,
+)
+from icecream import ic
```

Ruff (0.8.2)

`icecream.ic` imported but unused

Remove unused import: `icecream.ic`

(F401)